README.md

```diff
@@ -42,37 +36,91 @@ func WriteReader(path string, input io.ReadSeeker) error
 ```
 WriteReader Calls DefaultClient's WriteReader method.
 
-#### type Client
+#### type AWSClient
```

Comment:
It seems weird to call this an AWSClient when it supports s3 and the file system... not sure what a better name is, though.